.solhint.json

```diff
@@ -1,13 +1,32 @@
 {
     "extends": "solhint:default",
     "rules": {
-        "indent": ["error", 4],
-        "quotes": ["error", "double"],
-        "max-line-length": ["error", 129],
-        "function-max-lines": ["error", 55],
-        "max-states-count": ["error", 18],
+        "indent": [
+            "error",
+            4
+        ],
+        "quotes": [
+            "error",
+            "double"
+        ],
+        "max-line-length": [
+            "error",
+            156
```

Comment:
This is way to wide! I think this should be reverted back to 129 (which was already quite graceful).

Reply:
Unfortunately, I the prettier is not able to format everything with max-line-length of 129. This would require another issue in their github repo. Hence, I would like to stick with this wide max-line-length in order to use the prettier at all.

Reply:
I guess. although prettier is making our code less than 100 anyway... Seems fine to merge then.